spyder/utils/syntaxhighlighters.py

```diff
@@ -1230,6 +1220,37 @@ def highlightBlock(self, text):
             self.highlight_spaces(text)
 
 
+class PythonLoggingLexer(RegexLexer):
+    """
+    A lexer for logs generated by the Python builtin 'logging' library.
+
+    Taken from
+    https://bitbucket.org/birkenfeld/pygments-main/pull-requests/451/
+    add-python-logging-lexer

```

Comment:
The link should not be split between lines, even if it goes over 79 characters.

Probably is short/trivial enough that it doesn't need a mention in NOTICE.txt, but its up to you,

Reply:
It's code that was never merged to Pygments, so I don't think it deserves an entry in Notice.txt

Reply:
Legally that doesn't really change anything, but practically it might.